<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QMainWindow" name="Wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard for HikNetHextractor https://github.com/surfzoid/HikNetExtractor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Where to store the python files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>139</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PathED">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>641</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>the ip or domain name of the device</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserEd">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter the user name</string>
    </property>
    <property name="text">
     <string>admin</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="IpEd">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>the ip or domain name of the device</string>
    </property>
    <property name="text">
     <string>192.168.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Ip</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Pass</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnAdd">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>500</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Save and lose</string>
    </property>
    <property name="text">
     <string>Add/Modif</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PassEd">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter the user password</string>
    </property>
    <property name="text">
     <string>pass</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnCancel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Close without saving</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CamNameED">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>the ip or domain name of the device</string>
    </property>
    <property name="text">
     <string>MyHikCam</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Device</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Http port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PortHttpEd">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter the http port for ISAPI</string>
    </property>
    <property name="text">
     <string>800</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>101 first stream, 102 second stream, etc</string>
    </property>
    <property name="text">
     <string>Channel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ChannelEd">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>101 first stream, 102 second stream, etc</string>
    </property>
    <property name="text">
     <string>101</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>end with &quot;/&quot;</string>
    </property>
    <property name="text">
     <string>SavePath</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SavePathEd">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>end with &quot;/&quot;</string>
    </property>
    <property name="text">
     <string>~/</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>DaysToKeep</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DaysToKeepEd">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="UtcTOfset">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>129</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>if &quot;yes&quot;, UTC time offset of the computer is aply to end and start time of the file name.</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>UtcTOfset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSavePath">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnLoad">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>139</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
